steam-buddy

```diff
@@ -7,7 +7,13 @@ import shutil
 import flathub
 from bottle import route, request, static_file, run, template, redirect, response
 
-DATA_DIR='/var/lib/steam-buddy'
+if os.geteuid() == 0:
+	HOST_PORT = 80
+	DATA_DIR = '/var/lib/steam-buddy'
+else:
+	HOST_PORT = 8080
+	DATA_DIR = os.path.join(os.path.expanduser('~'), '.local/share/steam-buddy')
```

Comment:
we need to honour the `XDG_DATA_HOME` environment variable as well. `~/.local/share` is only the default location. See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
Good point, it now uses the ``XDG_DATA_HOME`` when possible, otherwise it falls back to ``~/.local/share``.